### PR TITLE
feat: change message structure

### PR DESCRIPTION
**Pull request by AtharvaVaidya**

### CODE REVIEW

File: src/hooks/useApi.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import useAuth from "@/hooks/useAuth";
 
-const backendUrl = process.env.NEXT_PUBLIC_BACKEND_URL || "http://0.0.0.0:8080/";
+const backendUrl = "http://0.0.0.0:8080/"//process.env.NEXT_PUBLIC_BACKEND_URL || "http://0.0.0.0:8080/";
```
Comment (AtharvaVaidya) [inline src/hooks/useApi.ts:3]: Need to revert this

File: src/components/History/History.tsx [hunk 2]
```diff
@@ -29,12 +30,25 @@ const History: React.FC<HistoryProps> = ({
...
-        type: message.role,
-        content: message.content
-      } as Message
+      const baseMessage: BaseMessage = {
+        role: message.role,
+        content: message.content,
+      };
+
+      if (message.role === 'user') {
+        console.log(message)
```
Comment (sam5pell) [inline src/components/History/History.tsx:41]: Flagging this log in case it was accidentally left in :)

**Comment by sam5pell** [comment]
When I ran it without backend running it kept retrying to get the image
image

File: src/components/History/History.tsx (hunk 2 above)
Comment (AtharvaVaidya) [reply src/components/History/History.tsx:41]: Oh yeah, thanks for flagging!

**Comment by AtharvaVaidya** [comment]
Gotcha, need to add error handling here. In the meantime I'm working on this PR in parallel for the backend: https://github.com/highlight-ing/highlight/pull/1151/files

File: src/components/Messages/Message.tsx [hunk 3]
```diff
@@ -15,10 +16,11 @@ import CodeBlock from "@/components/Messages/CodeBlock";
 const hasAttachment = (message: UserMessage) => {
   return (
     message.screenshot ||
-    message.clipboardText ||
+    message.clipboard_text ||
     message.window ||
     message.fileTitle ||
```
Comment (sam5pell) [inline src/components/Messages/Message.tsx:21]: This is nitpicky but if we're using snake case for `clipboard_text`, `image_url`, etc. can you change `fileTitle` to `file_title`.

Comment (AtharvaVaidya) [reply src/components/Messages/Message.tsx]: Done!

**Pull request merged**